src/products.hbs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+{{> partials/head}}
+<body>
+  {{> partials/header}}
+  <main>
+    <div class="loader" id='loader'>
+    </div>
+    <div class='product container' id='products-list-container' >
```

Comment:
Why do we need id? If there are 2 product list component then it will break and fail on W3C